deepchem/models/tests/test_mat.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import tempfile
+import numpy as np
+import deepchem as dc
+from deepchem.feat import MATFeaturizer
+from deepchem.models.torch_models import MATModel
+
+try:
+  import torch
+  has_torch = True
```

Comment:
It looks like this isn't being used in the fille.

Reply:
I have removed it now.